main.go

```diff
@@ -74,6 +74,18 @@ func main() {
 	}
 	// +kubebuilder:scaffold:builder
 
+	setupLog.Info("Registering Components.")
```

Comment:
We should consider a style guide for log messages, so they're consistent. I notice this uses caps and punctuation, but line 89 doesn't. I like keeping them stripped down, myself, but could go either way. Thoughts?

Reply:
This was added by me and the one in line 89 was added by kubebuilder. I'll make it consistent with the one added by kubebuilder.